buildpipeline/Core-Setup-Windows-BT.json

```diff
@@ -60,7 +60,62 @@
       "enabled": true,
       "continueOnError": false,
       "alwaysRun": false,
-      "displayName": "Build",
+      "displayName": "Install Signing Plugin",
```

Comment:
The names of these .json files are now super confusing.

`Core-Setup-Signing-Windows-BT.json` - does signing
`Core-Setup-Windows-BT.json` - also does signing